src/MasterPeace/Bundle/UpReadBundle/Controller/HomePageController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace MasterPeace\Bundle\UpReadBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class HomePageController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function redirectUserAction()
+    {
+        $response = null;
+
+        if ($this->getUser()->isTeacher()) {
```

Comment:
tiesiog jei studentas returnink i student.html.twig antraip i teacher - nereik to admin jum